src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -265,45 +260,72 @@
     <DropUnsignedFile Include="$(OutDir)\Microsoft.DebugEngineHost.pdb" />
-    <DropUnsignedFile Include="$(OutDir)\osxlaunchhelper.scpt" />
-    <DropUnsignedFile Include="@(RequiredNetFX46FacadeAssemblies)" />
-    <DropUnsignedFile Include="$(OutDir)\OpenFolderSchema.json" />
-  </ItemGroup>
-  <ItemGroup Condition="'$(IsCoreCLR)' == 'true'">
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.Android.natvis" />
+    <DropSignedFile Include="$(OutDir)\Microsoft.AndroidDebugLauncher.dll" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.AndroidDebugLauncher.pdb" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.AndroidDebugLauncher.pkgdef" />
+    <DropSignedFile Include="$(OutDir)\Microsoft.IOSDebugLauncher.dll" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.IOSDebugLauncher.pdb" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.IOSDebugLauncher.pkgdef" />
+    <DropSignedFile Include="$(OutDir)\Microsoft.JDbg.dll" />
+    <DropUnsignedFile Include="$(OutDir)\Microsoft.JDbg.pdb" />
     <DropSignedFile Include="$(OutDir)\Microsoft.SSHDebugPS.dll" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.SSHDebugPS.pdb" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.SSHDebugPS.pkgdef" />
+    <DropUnsignedFile Include="$(OutDir)\OpenFolderSchema.json" />
+    <DropUnsignedFile Include="@(RequiredNetFX46FacadeAssemblies)" />
   </ItemGroup>
   <ItemGroup Condition="'$(IsCoreCLR)' == 'false'">
+    <DropUnsignedFile Include="$(OutDir)\osxlaunchhelper.scpt" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.MICore.dll.mdb" />
     <DropUnsignedFile Include="$(OutDir)\Microsoft.MIDebugEngine.dll.mdb" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\MICore\MICore.csproj">
+      <Project>{54c33afa-438d-4932-a2f0-d0f2bb2fadc9}</Project>
+      <Name>MICore</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup;BuiltProjectOutputGroupDependencies;GetCopyToOutputDirectoryItems;SatelliteDllsProjectOutputGroup;</IncludeOutputGroupsInVSIX>
+    </ProjectReference>
+    <ProjectReference Include="..\MIDebugEngine\MIDebugEngine.csproj">
+      <Project>{a8b0230c-7806-407c-af18-54b2ce21275e}</Project>
+      <Name>MIDebugEngine</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup;BuiltProjectOutputGroupDependencies;GetCopyToOutputDirectoryItems;SatelliteDllsProjectOutputGroup;</IncludeOutputGroupsInVSIX>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup Condition="'$(IsCoreCLR)' == 'false'">
+    <ProjectReference Include="..\DebugEngineHost.VSCode\DebugEngineHost.VSCode.csproj">
+      <Project>{81de2423-fb5e-4069-b3c5-4c13ce76dc0a}</Project>
+      <Name>DebugEngineHost.VSCode</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\OpenDebugAD7\OpenDebugAD7.csproj">
+      <Project>{d582c0b6-032f-4686-b01d-eb55694059cb}</Project>
+      <Name>OpenDebugAD7</Name>
+    </ProjectReference>
+  </ItemGroup>
```

Comment:
Since you aren't singing these guys from this project, any reason you want this?

Reply:
Eventually i wanted to move it here. I'll remove it for now.